x-pack/plugins/uptime/e2e/journeys/utils.ts

```diff
@@ -25,3 +24,12 @@ export async function loginToKibana({ page }: { page: Page }) {
 
   await waitForLoadingToFinish({ page });
 }
+
+export const byTestId = (testId: string) => {
+  return `[data-test-subj=${testId}]`;
+};
+
+export const assertText = async ({ page, text }: { page: Page; text: string }) => {
+  await page.waitForSelector(`text=${text}`);
+  expect(await page.$(`text=${text}`)).toBeTruthy();
```

Comment:
Do we need this method to contain these two lines? Wouldn't the first already cause the test to fail in case the selector didn't appear before the timeout?

Reply:
i think waitForSelector doesn't throw an error, it returns null